newrelic/data_source_newrelic_synthetics_secure_credential_test.go

```diff
@@ -8,13 +8,11 @@ import (
 	"strings"
 	"testing"
 
-	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
 )
 
 func TestAccNewRelicSyntheticsSecureCredentialDataSource_Basic(t *testing.T) {
-	rand := acctest.RandString(5)
-	rName := fmt.Sprintf("tf_test_%s", rand)
+	rName := "TF_ACC_TEST"
```

Comment:
Should we keep the random string generation for creating unique resource names?   This keeps us from having to create and keep track of a bunch of test resources that could accidentally get deleted and cause tests to fail.

One thing is for sure though, it looks like we need to fix the "cleanup" cron job that was supposed to clear out any leftover `tf_test*` resources. It looks like it may not be running anymore because we have approximately 2600 hosts prefixed with `tf_test_*`  😆  This is due to our integration tests leaving some isolated resources when unexpected errors occur.